### PR TITLE
City of bakersfield

### PR DESCRIPTION
**Pull request by chachasikes**
Added residential & fire water rates for City of Bakersfield.

### CODE REVIEW

File: full_utility_rates/California/City of Bakersfield - 57/cob-2017-06-01.owrs [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+         5/8"  : 9.88
+         3/4"  : 9.88
+         1"    : 15.06
+         1_1/2": 22.98
+         2"    : 31.28
+         3"    : 57.80
+         4"    : 84.02
+         6"    : 151.00
+         8"    : 232.12
+         10"   : 319.95
```
Comment (vincent-hebble) [inline full_utility_rates/California/City of Bakersfield - 57/cob-2017-06-01.owrs]: Need to add bill structure and commodity_charge if there is one such as 

if there is a commodity_charge
bill: "commodity_charge+service_charge"

or if only service_charge
bill: "service_charge"

**Comment by patwater** [comment]
@ArashSeif could you please review and update as needed

**Comment by ArashSeif** [comment]
@patwater updated rates are added under branch #293.

**Comment by ArashSeif** [comment]
@patwater this branch is also updated and fixed

**Pull request merged**